core/templates/domain/utilities/language-util.service.ts

```diff
@@ -149,25 +149,20 @@ export class LanguageUtilService {
     return language ? language.description : null;
   }
 
-  getContentLanguageDirection(languageCode: string): string {
-    let languageDirection = 'auto'; // Default auto for invalid languageCode
-    this.SUPPORTED_CONTENT_LANGUAGES.forEach((lang: ContentLanguage) => {
-      if (lang.code === languageCode) {
-        languageDirection = lang.direction;
+  getLanguageDirection(languageCode: string): string {
+    for (let i = 0; i < this.SUPPORTED_CONTENT_LANGUAGES.length; i++) {
+      const language = this.SUPPORTED_CONTENT_LANGUAGES[i];
+      if (language.code === languageCode) {
+        return language.direction;
       }
```

Comment:
I think this traditional loop can be simplified somewhat using something like [Array.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find).

Reply:
Thanks for the recommendation! I had been trying to figure out a way to clean it up with a forEach, I forgot about find.
Done.